### ISSUE TITLE
Remove dependency on WSL?

### ISSUE THREAD

**Issue by strich** [microsoft/infersharp#189]
Infer looks really nice, however it does not appear to be something we can add into our standard pipelines amongst the team as it has pretty heavily dependencies on things like WSL. Ideally this can be added to a C# project and run on any machine without external dependencies.

**Comment by matjin** [comment]
Yes -- this is one of our currently most-requested items right now. We are looking into this in partnership with the Infer team at Meta!
